Package.swift

```diff
@@ -9,23 +9,25 @@ let package = Package(
     products: [
         .library(
             name: "ZXingCppWrapper",
-            type: .static,
```

Comment:
Short remark: this should lead to either generate it statically or dynamically depending on the target, cf. [library(name:type:targets:)](https://developer.apple.com/documentation/packagedescription/product/library(name:type:targets:)).